doc/generate-api-docs.sh

```diff
@@ -51,10 +51,29 @@ LIBS_SORTED=$(
 
 for LIB in $LIBS_SORTED; do
     cd $LIB
-    ARGS=""
-    if FEATURES="$(cargo read-manifest|jq -r '.metadata.docs.rs.features | join(",")' 2> /dev/null)"; then
-        ARGS="--features $FEATURES"
+    version=$(git tag --sort=taggerdate | grep $LIB'_' | tail -n1 | cut -d'_' -f2 | cut -d'v' -f2)
+    dependency=$LIB' = { version = "'$version'"'
```

Comment:
You'll want to use an exact version specification: `version = "='$version'"`

Reply:
Done.